java/jtraverser2/src/main/java/jtraverser/TreeManager.java

```diff
@@ -607,12 +607,12 @@ public void ancestorRemoved(final AncestorEvent ae) {/**/}
 				});
 				final Object[] ob = {"[ssh://][<user>@]<hostname>[:<port>]", provider};
 				final int result = JOptionPane.showConfirmDialog(JOptionPane.getRootFrame(), ob, "Open new connection", JOptionPane.OK_CANCEL_OPTION, JOptionPane.PLAIN_MESSAGE);
-				if(result == JOptionPane.OK_OPTION) new Thread(new Runnable(){
+				if(result == JOptionPane.OK_OPTION) new Thread(null,new Runnable(){
```

Comment:
If I understand correctly, this sets the ThreadGroup to null, what does that change?
Will this break on older versions of Java or anything?

Reply:
it was null before but as i want to set the third argument "name" i need to explicidly set the first one because there was not two arg version of the constructor

Reply:
Oh, I missed the new parameter at the end